recipes-devtools/cargo/cargo.inc

```diff
@@ -1,6 +1,6 @@
 INHIBIT_CARGO_DEP = "1"
```

Comment:
Seems like `cargo_util` ignores this? Is it unneeded? I included it initially because cargo would use a cargo snapshot to build itself, and I believed that I needed to avoid adding `cargo-native` unconditionally to avoid a dependency loop (though I don't think I ever checked omitting it).

If `bitbake` is happy with this current set of changes, you should be able to remove this unused variable.